addons/l10n_it_edi_withholding/models/account_move.py

```diff
@@ -19,10 +19,11 @@ class AccountMove(models.Model):
     @api.depends('amount_total_signed')
     def _compute_amount_extended(self):
         for move in self:
-            totals = dict(vat=0.0, withholding=0.0, pension_fund=0.0)
+            totals = {None: 0.0, 'vat':0.0, 'withholding': 0.0, 'pension_fund': 0.0}
```

Comment:
When is it None?  No other way to handle that?

Reply:
It's the case of Split Payment, when we have a group of 2 taxes, one is VAT tax, the other is the negative workaround. It's implemented like this to have taxes belonging to 2 different tax groups, so that we have separated totals.